pkg/controller/service/servicecontroller.go

```diff
@@ -670,8 +670,7 @@ func (s *ServiceController) lockedUpdateLoadBalancerHosts(service *api.Service,
 		return nil
 	}
 
-	message := "error updating loadbalancer with new hosts: " + err.Error()
-	s.eventRecorder.Event(service, "LoadBalancerUpdateFailed", message)
+	s.eventRecorder.Eventf(service, "LoadBalancerUpdateFailed", "Error updating load balancer with new hosts: %v", err)
```

Comment:
Including actual hosts may make it less likely that identical events will be coalesced.  Also, the hosts were not passed to the function that had the error, so they can't really be part of the Reason.